app/src/androidTest/java/com/github/se/polyfit/ui/screen/CreatePostTest.kt

```diff
@@ -154,5 +157,9 @@ class CreatePostTest : TestCase() {
         assertIsEnabled()
       }
     }
+
+    ComposeScreen.onComposeScreen<CreatePostBottomBar>(composeTestRule) {
+      composeTestRule.onNodeWithTag("BackButton").assertExists().assertIsDisplayed().performClick()
+    }
```

Comment:
This could be included in the `ComposeScreen.onComposeScreen<CreatePostBottomBar>(composeTestRule) {` above on line 153